libraries/AP_DDS/dds_xrce_profile.xml

```diff
@@ -114,6 +114,29 @@
       </historyQos>
     </topic>
   </data_writer>
+  <topic profile_name="imu__t">
+    <name>rt/ap/imu/data</name>
+    <dataType>sensor_msgs::msg::dds_::Imu_</dataType>
+    <historyQos>
+      <kind>KEEP_LAST</kind>
+      <depth>20</depth>
+    </historyQos>
+  </topic>
+  <data_writer profile_name="imu__dw">
+    <topic>
+      <kind>NO_KEY</kind>
+      <name>rt/ap/imu/data</name>
```

Comment:
```suggestion
      <name>rt/ap/imu/experimental/data</name>
```

Reply:
Make sure to change the topic name in both the `topic` and `data_writer` objects. It likely didn't work when you tried because you only did one of them.

Proposed topic in the XML file: `rt/ap/experimental/imu`

Results:
```
$ ros2 topic list
/ap/battery/battery0
/ap/clock
/ap/cmd_gps_pose
/ap/cmd_vel
/ap/exp_imu
/ap/experimental/imu
/ap/geopose/filtered
/ap/joy
/ap/navsat/navsat0
/ap/pose/filtered
/ap/tf
/ap/tf_static
/ap/time
/ap/twist/filtered
/parameter_events
/rosout
```

Reply:
This change has been committed. It is working properly now